### PR TITLE
Anim14.sh. Error in proces

### PR DESCRIPTION
**Pull request by Esteban82**
There was an error in the processing of the ndk file which resulted in a wrong display of the planes of the focal mechanisms. I update the processing (line 13) and convention (-Sc instead of -Sd). Then I must update meca.gmt in the server. The youtube animation must be corrected.

### CODE REVIEW

**Comment by PaulWessel** [comment]
OK, just post the meca.gmt file here and either @meghanrjones or I will take care of the upload.

**Comment by Esteban82** [comment]
Ok. Here is the meca file compressed. 
[meca.zip](https://github.com/GenericMappingTools/gmt/files/6418780/meca.zip)

**Comment by seisman** [comment]
Perhaps better to have a longer name for this data? What about "GCMT_1976-2017_meca.gmt"?

**Comment by Esteban82** [comment]
It sounds good, seisman.

Also, I just read in the header that the data is from western Argentina and central Chile but in reality it has data from all over the world from that period of time. So maybe we could clarify that or filter the data by area to reduce its size.

**Comment by PaulWessel** [comment]
Yes, I think using the region to limit the data is good - we want the test files to be small.

**Comment by Esteban82** [comment]
Ok. Here is the new file. 

In its header I add this line:
# gmt select meca.gmt -R-75.1/-63/-34.44/-30.35 > GCMT_1976-2017_meca.gmt
[GCMT_1976-2017_meca.zip](https://github.com/GenericMappingTools/gmt/files/6418977/GCMT_1976-2017_meca.zip)

File: doc/examples/anim14/anim14.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # The original data file jan76_dec17.ndk can be downloaded and rearranged to a gmt format with the followings commands:
 # URL="https://www.ldeo.columbia.edu/~gcmt/projects/CMT/catalog/jan76_dec17.ndk"
 # gmt which $URL -G 
-# gawk '/^PDE/ {Date=$2; Time=$3; Lat=$4; Long=$5; Depth=$6; getline; Name=$1; getline; getline; Exp=$1; getline; strike1=$12; dip1=$13; rake1=$14; strike2=$15; dip2=$16; rake2=$17; print Long, Lat, Depth, strike1, dip1, rake1, strike2, dip2, rake2, Exp, Date "T" Time, Name}' jan76_dec17.ndk | sed 's/\//-/g' > meca.gmt
+# gawk '/^PDE/ {Date=$2; Time=$3; Lat=$4; Long=$5; Depth=$6; getline; Name=$1; getline; getline; Exp=$1; getline; mant=$11; strike1=$12; dip1=$13; rake1=$14; strike2=$15; dip2=$16; rake2=$17; print Long, Lat, Depth, strike1, dip1, rake1, strike2, dip2, rake2, Exp, Date "T" Time, Name}' jan76_dec17.ndk | sed 's/\//-/g' > meca.gmt
```
Comment (meghanrjones) [inline doc/examples/anim14/anim14.sh:13]: This is the version in the header for GCMT_1976-2017_meca.gmt. Please check which is correct.
```suggestion
# gawk '/^PDE/ {Date=$2; Time=$3; Lat=$4; Long=$5; Depth=$6; getline; Name=$1; getline; getline; Exp=$1; getline; mant=$11; strike1=$12; dip1=$13; rake1=$14; strike2=$15; dip2=$16; rake2=$17; print Long, Lat, Depth, strike1, dip1, rake1, strike2, dip2, rake2, mant, Exp, Date "T" Time, Name}' jan76_dec17.ndk | sed 's/\//-/g' > meca.gmt
```

**Comment by meghanrjones** [comment]
I will build the animation and replace the current version on YouTube. Can you please update the meca.gmt reference in line 8 as well @Esteban82?

**Comment by meghanrjones** [comment]
@Esteban82, can you please check that the YouTube video looks good: https://youtu.be/Wk58r72g_nk? If so, can you please change the youtube reference `9D1qANCoptE` to `Wk58r72g_nk` in `doc/examples/anim14/anim14.sh`, `doc/rst/source/animations.rst`, and `doc/rst/source/animations/anim14.rst`?

**Comment by meghanrjones** [comment]
The YouTube link in `doc/rst/source/animations.rst` needs to be updated

File: doc/rst/source/animations.rst [hunk 2]
```diff
@@ -81,8 +81,7 @@ includes the animations) does not take excessive time.
 
   :doc:`/animations/anim14`
 
-.. toctree::
-   :hidden:
-   :glob:
+- .. youtube:: Wk58r72g_nk
+     :width: 100%
```
Comment (meghanrjones) [inline doc/rst/source/animations.rst:85]: The youtube link occurs before the file references, so `Wk58r72g_nk` needs to replace `9D1qANCoptE` on line 79 and the toctree in these lines should remain.
```suggestion
.. toctree::
   :hidden:
   :glob
```

**Review by meghanrjones — APPROVED** [review]

**Comment by meghanrjones** [comment]
Good to merge, @Esteban82

**Comment by meghanrjones** [comment]
One quick note about merging, you can shorten the merge commit message after clicking 'squash and merge' to one descriptive line for simple PRs like this one.

**Pull request merged**